site/docs/benchmarks.md

```diff
@@ -0,0 +1,110 @@
+<!--
+  - Licensed to the Apache Software Foundation (ASF) under one
+  - or more contributor license agreements.  See the NOTICE file
+  - distributed with this work for additional information
+  - regarding copyright ownership.  The ASF licenses this file
+  - to you under the Apache License, Version 2.0 (the
+  - "License"); you may not use this file except in compliance
+  - with the License.  You may obtain a copy of the License at
+  -
+  -   http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing,
+  - software distributed under the License is distributed on an
+  - "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  - KIND, either express or implied.  See the License for the
+  - specific language governing permissions and limitations
+  - under the License.
+  -->
+
+## Available Benchmarks and how to run them
+
```

Comment:
I would consider mentioning that it's probably not a good idea to run all of the benchmarks at once, because they run in the same JVM.

So doing something like `./gradlew :jmh` is likely to fail (or give really incorrect results). Maybe just mentioning that tests of interest should be run independently for a better representation?

Reply:
good point, updated the description